src/rules/ResourceDNTTerms.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * ResourceDNTTerms.js - rule to ensure that Do Not Translate terms have not been translated
+ *
+ * Copyright © 2023 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Rule, Result } from "i18nlint-common";
+import fs from "node:fs";
+
+/** Rule to ensure that Do Not Translate terms have not been translated;
+ * i.e., if a DNT term appears in source, it has to appear in target as well */
+class ResourceDNTTerms extends Rule {
+    /** @readonly */ name = "resource-dnt-terms";
+    /** @readonly */ description = "Ensure that Do Not Translate terms have not been translated.";
+    /** @readonly */ link = "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-dnt-terms.md";
+    /** @readonly */ ruleType = "resource";
+
+    /**
+     * @readonly
+     * @type {string[]}
+     */
+    _dntTerms = [];
+
+    get dntTerms() {
+        return [...this._dntTerms];
```

Comment:
Why do we need a getter for this array?

Reply:
That one I exposed on a whim to be able to unittest if terms were loaded correctly. I guess it's not really needed, so I'll just change the static parsing methods to be public and test them directly

Reply:
Changed in 1ea955f